packages/composites/synth-index/src/index.ts

```diff
@@ -1,7 +1,5 @@
 import { expose } from '@chainlink/ea-bootstrap'
 import { makeExecute } from './adapter'
-import { makeConfig } from './config'
 
-const NAME = 'SYNTH-INDEX'
-
-export = { NAME, makeConfig, makeExecute, ...expose(makeExecute()) }
```

Comment:
This is losing the `makeConfig` and `makeExecute` exports?

Reply:
Yeah, i dropped them since it looks like they're not being used